src/o365/todo/commands/list/list-set.ts

```diff
@@ -0,0 +1,105 @@
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import GraphCommand from '../../../base/GraphCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+  name: string;
+}
+
+class TodoListSetCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.LIST_SET}`;
+  }
+
+  public get description(): string {
+    return 'Updates a Microsoft To Do task list';
+  }
+
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    const body: any = {
+      displayName: args.options.name
+    };
+
+    const requestOptions: any = {
+      url: `${this.resource}/beta/me/todo/lists/${args.options.id}`,
+      headers: {
+        accept: 'application/json;odata.metadata=none',
+        'content-type': 'application/json'
+      },
+      body,
+      json: true
+    };
+
+    request
+      .patch(requestOptions)
+      .then((res: any): void => {
+        if (this.verbose) {
+          cmd.log(vorpal.chalk.green('DONE'));
+        }
+
+        cb();
+      }, (err: any) => this.handleRejectedODataJsonPromise(err, cmd, cb));
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.name = typeof args.options.name !== 'undefined';
+    telemetryProps.id = typeof args.options.id !== 'undefined';
+    return telemetryProps;
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-i, --id <id>',
+        description: `The ID of the list to update`
+      },
+      {
+        option: '-n, --name <name>',
```

Comment:
Let's update this property to `newName`. In the future we want to allow users to retrieve resources by their name, so to avoid breaking changes in the future, let's pick a neutral name for this option.